src/cairoUtilFuncGen/storage/mappingIndexAccess.ts

```diff
@@ -229,8 +228,8 @@ export class MappingIndexAccessGen extends CairoUtilFuncGenBase {
           }
       `,
       functionsCalled: [
-        // this.requireImport(['array'], 'ArrayTrait'),
-        // this.requireImport(['warplib', 'string_hash'], 'string_hash'),
+        this.requireImport(['array'], 'ArrayTrait'),
```

Comment:
Do you think this PR is a good place to start adding imports for Cairo1 code? 
```suggestion
        this.requireImport(...ARRAY_TRAIT),
```

Reply:
Yeah I agree, let's just separate them by the comment line from the old imports